glrd/query.py

```diff
@@ -7,6 +7,7 @@
 import tabulate
 from glrd.util import *
 import sys
+import logging
```

Comment:
It's always a good idea to sort by alphabet.

Reply:
:+1: